pandas/tests/reshape/merge/test_merge.py

```diff
@@ -1491,9 +1491,12 @@ def test_different(self, dtype):
         result = merge(left, right, on="A")
         assert is_object_dtype(result.A.dtype) or is_string_dtype(result.A.dtype)
 
+    @pytest.mark.parametrize(
+        "d1", [np.int64, np.int32, np.intc, np.int16, np.int8, np.uint8, np.uintc]
+    )
     @pytest.mark.parametrize("d2", [np.int64, np.float64, np.float32, np.float16])
-    def test_join_multi_dtypes(self, any_int_numpy_dtype, d2):
```

Comment:
Could you keep this test the same and add a new separate test for `np.uintc`?

Reply:
ok sure